File: string-manipulation/alphabetize/JavaScript/alphabetize.js
// Write a JavaScript function that returns a passed string with letters in alphabetical order.
// Example string : 'webmaster'
// Expected Output : 'abeemrstw'
// Assume punctuation and numbers symbols are not included in the passed string.


function alphabetize(string) {
  var sentenceArr = string.split(' ')
  var finalSentence = []
  sentenceArr.forEach(function(entry) {
    var newArr = entry.split('').sort();
    finalSentence.push(newArr.join(''));
  });
  console.log(finalSentence.join(' '))
};

alphabetize("webmaster is here")

File: string-manipulation/alphabetize/Ruby/alphabetize.rb
class Alphabetize
  def alpha(string)
    str_arr = string.gsub(' ', '').split('')
    str_arr.sort.join
  end
end

puts Alphabetize.new.alpha("wee let's go! woohoo")
puts Alphabetize.new.alpha("gfedcba")
